### ISSUE TITLE
Model::offsetExists() throws UnknownPropertyException when property doesn’t exsit

### ISSUE THREAD

**Issue by brandonkelly** [yiisoft/yii2#12030]
### What steps will reproduce the problem?

1. Create a class that extends yii\base\Model

        class MyModel extends yii\base\Model
        {
        }

2. Instantiate the model and check if a nonexistant property isset:

        $obj = new MyModel();
        $isSet = isset($obj->foo);

### What is the expected result?

Model::offsetExists() should return `false` (which is called internally when you call isset()).

### What do you get instead?

A yii\base\UnknownPropertyException is thrown with the message:

> Getting unknown property: MyModel::foo

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.9
| PHP version      | 	5.6.13-1+deb.sury.org~trusty+3
| Operating system | Linux homestead 3.19.0-25-generic #26~14.04.1-Ubuntu SMP Fri Jul 24 21:16:20 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

**Comment by klimov-paul** [comment]
Following code works absolutely fine:

```php
$obj = new MyModel();
$isSet = isset($obj->foo);
```

I assume you refer to `ArrayAccess` isset syntax:

```php
$obj = new MyModel();
$isSet = isset($obj['foo']); // throw an exception
```

**Comment by klimov-paul** [comment]
Resolved by commit 8de6a5117ca4bd95333cbd6f3d0bbe0cfdd800b1

**Comment by brandonkelly** [comment]
Oops, sorry for the bad example! Yes that should have been `isset($obj['foo']);`. Thanks for fixing!
